pkg/agent/orm/manager.go

```diff
@@ -102,6 +122,7 @@ func NewManager(socketPath string, emitter metrics.MetricEmitter, metaServer *me
 	}
 
 	m := &ManagerImpl{
+		mode:       workMode(config.ORMWorkMode),
```

Comment:
all the work mode initialization move to `initORMWorkMode` may be better.

Reply:
sure, good point